src/libs/actions/IOU.js

```diff
@@ -99,26 +116,75 @@ function startLoadingAndResetError() {
 }
 
 /**
- * Creates IOUSplit Transaction
+ * Request money from another user
  *
- * @param {Object} params
- * @param {Number} params.amount
- * @param {String} params.comment
- * @param {String} params.currency
- * @param {String} params.debtorEmail
+ * @param {Object} report
+ * @param {Array} participants
+ * @param {Number} amount
+ * @param {String} currency
+ * @param {String} recipientEmail
+ * @param {String} debtorEmail
+ * @param {String} comment
  */
-function createIOUTransaction(params) {
-    startLoadingAndResetError();
-    DeprecatedAPI.CreateIOUTransaction(params)
-        .then((response) => {
-            if (response.jsonCode !== 200) {
-                processIOUErrorResponse(response);
-                return;
-            }
-
-            getIOUReportsForNewTransaction([response]);
-            Navigation.navigate(ROUTES.getReportRoute(response.chatReportID));
-        });
+function requestMoney(report, participants, amount, currency, recipientEmail, debtorEmail, comment) {
+    const chatReport = lodashGet(report, 'reportID', null) ? report : Report.buildOptimisticChatReport(participants);
+    const optimisticTransactionID = NumberUtils.rand64();
+    let IOUReport;
+    if (chatReport.hasOutstandingIOU) {
+        IOUReport = iouReports[`${ONYXKEYS.COLLECTION.REPORT_IOUS}${chatReport.iouReportID}`];
+        IOUReport.total += amount;
+    } else {
+        IOUReport = ReportUtils.buildOptimisticIOUReport(recipientEmail, debtorEmail, amount, chatReport.reportID, currency, 'en');
+    }
+    const optimisticReportAction = ReportUtils.buildOptimisticIOUReportAction('create', amount, 'comment', currency, '', optimisticTransactionID, IOUReport.reportID, debtorEmail);
```

Comment:
This should be the parameter, not a string right?
```suggestion
    const optimisticReportAction = ReportUtils.buildOptimisticIOUReportAction('create', amount, comment, currency, '', optimisticTransactionID, IOUReport.reportID, debtorEmail);
```

Reply:
Well, 'comment' is a great comment! Fixing it 😆